script/dewebsockify

```diff
@@ -6,129 +6,32 @@
 use Mojo::Base -strict, -signatures;
 
 use Getopt::Long;
-use Mojo::Cookie::Request;
-use Mojo::IOLoop;
-use Mojo::IOLoop::Server;
-use Mojo::IOLoop::Stream;
-use Mojo::Log;
-use Mojo::UserAgent;
-
-sub usage ($r = 0) {
-    say 'Listens on a TCP port forwarding data to the specified websocket server
-    example: --websocketurl wss://... --listenport 590x --cookie "vmware_client=VMware; some_session=foobar" --insecure';
-    exit $r;
+use FindBin;
+use lib "$FindBin::Bin/.."; # Ensure OpenQA namespace is in @INC
+use OpenQA::Isotovideo::Dewebsockify;
+
+OpenQA::Isotovideo::Dewebsockify::main(parse_args()) unless caller();
```

Comment:
Just mentioning: since you moved the code to a real module, this trick to be able to unit test certain functions in a script is not necessary.
It won't hurt to keep that line.